docker-compose.yml

```diff
@@ -101,6 +101,38 @@ services:
       - 5432:5432
     volumes:
       - /var/lib/postgresql/data
+  rbac:
+    image: quay.io/cloudservices/rbac:latest
+    restart: on-failure
+    environment:
+      - REDIS_PORT=6379
+      - REDIS_HOST=redis
+      - DATABASE_HOST=db
+      - DATABASE_PORT=5432
+      - DATABASE_NAME=insights
+      - DATABASE_USER=insights
+      - DATABASE_PASSWORD=insights
+      - API_PATH_PREFIX=/api/rbac
+      - DEVELOPMENT=True
+    depends_on:
+      - db
+      - redis
+    links:
+      - db
+      - redis
+    ports:
+      - 9002:8080
+  build-rbac-db:
+    image: quay.io/cloudservices/rbac:latest
+    environment:
+      - TENANT_PARALLEL_MIGRATION_MAX_PROCESSES=4
+      - TENANT_PARALLEL_MIGRATION_CHUNKS=2
+    depends_on:
+      - rbac
+    links:
+      - rbac
+    entrypoint: /bin/sh
+    command: -c './rbac/manage.py migrate_schemas --executor=parallel'
```

Comment:
Will this make the API requests pass? :thinking: We should not break existing functionality, even in development.

Reply:
Please also check if you need to map the volumes here instead (for seeding).